init.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Clean previous run data.
+rm -f admin_data data_key
+
+docker-compose stop
+if [ $? -eq 0 ]; then
```

Comment:
This logic shows up a lot in this script.  What about abstracting it out into a method:
```sh
function execute() {
  eval $1
  if [ $? -eq 0 ]; then
    echo "$1... OK"
  else
    echo "$1... FAIL"
  fi
}
```
Which will allow us to call these methods with:
```sh
execute 'docker-compose stop'
```

Reply:
Oops it was meant to be only for dev, but I think it can be useful - I added that function and completed all 4 Units